### PR TITLE
Add documentation for packages - 2022-04-30

### PR DESCRIPTION
**Pull request by potiuk**

### CODE REVIEW

**Review by jedcunningham — APPROVED** [review]

**Comment by jedcunningham** [comment]
Btw, this is missing (YANKED) on the yanked versions. We might need a way to mark them so it gets generated that way instead of having to do it manually each time.

File: docs-archive/apache-airflow-providers-cncf-kubernetes/4.0.1/index.html [hunk 1]
```diff
@@ -0,0 +1,1374 @@
...
+</div>
+<div class="section" id="id3">
+<h4>Bug Fixes<a class="headerlink" href="#id3" title="Permalink to this headline">¶</a></h4>
+<ul class="simple">
+<li><p><code class="docutils literal notranslate"><span class="pre">Stop</span> <span class="pre">crashing</span> <span class="pre">when</span> <span class="pre">empty</span> <span class="pre">logs</span> <span class="pre">are</span> <span class="pre">received</span> <span class="pre">from</span> <span class="pre">kubernetes</span> <span class="pre">client</span> <span class="pre">(#22566)</span></code></p></li>
+</ul>
+</div>
+</div>
+<div class="section" id="id4">
+<h3>3.1.2<a class="headerlink" href="#id4" title="Permalink to this headline">¶</a></h3>
```
Comment (jedcunningham) [inline docs-archive/apache-airflow-providers-cncf-kubernetes/4.0.1/index.html:710]:
```suggestion
<h3>3.1.2 <span class="pre" style="color: red">(YANKED)</span><a class="headerlink" href="#id4" title="Permalink to this headline">¶</a></h3>
```

File: docs-archive/apache-airflow-providers-cncf-kubernetes/4.0.1/index.html [hunk 2]
```diff
@@ -0,0 +1,1374 @@
...
+</div>
+<div class="section" id="misc">
+<h4>Misc<a class="headerlink" href="#misc" title="Permalink to this headline">¶</a></h4>
+<ul class="simple">
+<li><p><code class="docutils literal notranslate"><span class="pre">Remove</span> <span class="pre">RefreshConfiguration</span> <span class="pre">workaround</span> <span class="pre">for</span> <span class="pre">K8s</span> <span class="pre">token</span> <span class="pre">refreshing</span> <span class="pre">(#20759)</span></code></p></li>
+</ul>
+</div>
+</div>
+<div class="section" id="id6">
+<h3>3.1.1<a class="headerlink" href="#id6" title="Permalink to this headline">¶</a></h3>
```
Comment (jedcunningham) [inline docs-archive/apache-airflow-providers-cncf-kubernetes/4.0.1/index.html:726]:
```suggestion
<h3>3.1.1 <span class="pre" style="color: red">(YANKED)</span><a class="headerlink" href="#id6" title="Permalink to this headline">¶</a></h3>
```

File: docs-archive/apache-airflow-providers-cncf-kubernetes/4.0.1/index.html [hunk 3]
```diff
@@ -0,0 +1,1374 @@
...
+<h4>Misc<a class="headerlink" href="#id10" title="Permalink to this headline">¶</a></h4>
+<ul class="simple">
+<li><p><code class="docutils literal notranslate"><span class="pre">Support</span> <span class="pre">for</span> <span class="pre">Python</span> <span class="pre">3.10</span></code></p></li>
+<li><p><code class="docutils literal notranslate"><span class="pre">Fix</span> <span class="pre">Kubernetes</span> <span class="pre">example</span> <span class="pre">with</span> <span class="pre">wrong</span> <span class="pre">operator</span> <span class="pre">casing</span> <span class="pre">(#21898)</span></code></p></li>
+<li><p><code class="docutils literal notranslate"><span class="pre">Remove</span> <span class="pre">types</span> <span class="pre">from</span> <span class="pre">KPO</span> <span class="pre">docstring</span> <span class="pre">(#21826)</span></code></p></li>
+</ul>
+</div>
+</div>
+<div class="section" id="id11">
+<h3>3.0.2<a class="headerlink" href="#id11" title="Permalink to this headline">¶</a></h3>
```
Comment (jedcunningham) [inline docs-archive/apache-airflow-providers-cncf-kubernetes/4.0.1/index.html:753]:
```suggestion
<h3>3.0.2 <span class="pre" style="color: red">(YANKED)</span><a class="headerlink" href="#id11" title="Permalink to this headline">¶</a></h3>
```

File: docs-archive/apache-airflow-providers-cncf-kubernetes/4.0.1/index.html [hunk 4]
```diff
@@ -0,0 +1,1374 @@
...
+<div class="section" id="id12">
+<h4>Bug Fixes<a class="headerlink" href="#id12" title="Permalink to this headline">¶</a></h4>
+<ul class="simple">
+<li><p><code class="docutils literal notranslate"><span class="pre">Add</span> <span class="pre">missed</span> <span class="pre">deprecations</span> <span class="pre">for</span> <span class="pre">cncf</span> <span class="pre">(#20031)</span></code></p></li>
+</ul>
+</div>
+</div>
+<div class="section" id="id13">
+<h3>3.0.1<a class="headerlink" href="#id13" title="Permalink to this headline">¶</a></h3>
```
Comment (jedcunningham) [inline docs-archive/apache-airflow-providers-cncf-kubernetes/4.0.1/index.html:762]:
```suggestion
<h3>3.0.1 <span class="pre" style="color: red">(YANKED)</span><a class="headerlink" href="#id13" title="Permalink to this headline">¶</a></h3>
```

**Comment by potiuk** [comment]
> Btw, this is missing (YANKED) on the yanked versions. We might need a way to mark them so it gets generated that way instead of having to do it manually each time.

Ah yeah good point @jedcunningham . I will add it.

**Pull request merged**